test/providers/provider_level_zero.cpp

```diff
@@ -2,11 +2,12 @@
 // Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#include <dlfcn.h>
-#include <level_zero/ze_api.h>
+#define NOMINMAX //workaround for std::numeric_limits
```

Comment:
I'm not sure if in-line comment with `define` is the best idea, pls move it up one line

Reply:
Done